toolchains/Toolchain-QNX7-QCC-Base.cmake

```diff
@@ -0,0 +1,79 @@
+set(CMAKE_SYSTEM_NAME QNX)
+
+set(CMAKE_SHARED_LIBRARY_PREFIX "lib")
+set(CMAKE_SHARED_LIBRARY_SUFFIX ".so")
+set(CMAKE_STATIC_LIBRARY_PREFIX "lib")
+set(CMAKE_STATIC_LIBRARY_SUFFIX ".a")
+
+if(CMAKE_HOST_WIN32)
+    set(HOST_EXECUTABLE_SUFFIX ".exe")
+endif()
+
+if(QNX_HOST)
+    set(ENV{QNX_HOST} "${QNX_HOST}")
+else()
+    set(QNX_HOST "$ENV{QNX_HOST}")
+    if("${QNX_HOST}" STREQUAL "")
+        message(FATAL_ERROR "QNX_HOST environment variable is not defined")
+    endif()
+    set(QNX_HOST "${QNX_HOST}" CACHE INTERNAL "")
+endif()
+message(STATUS "Using QNX_HOST ${QNX_HOST}")
+
+if(QNX_TARGET)
+    set(ENV{QNX_TARGET} "${QNX_TARGET}")
+else()
+    set(QNX_TARGET "$ENV{QNX_TARGET}")
+    if("${QNX_TARGET}" STREQUAL "")
+        message(FATAL_ERROR "QNX_TARGET environment variable is not defined")
+    endif()
+    set(QNX_TARGET "${QNX_TARGET}" CACHE INTERNAL "")
+endif()
+message(STATUS "Using QNX_TARGET ${QNX_TARGET}")
+
+if(NOT EXISTS "${QNX_TARGET}/etc/qversion")
+    message(FATAL_ERROR "The folder ${QNX_TARGET} is missing /etc/qversion and may not provide a QNX SDK")
+endif()
+
+file(READ "${QNX_TARGET}/etc/qversion" CMAKE_SYSTEM_VERSION)
+string(REGEX REPLACE "[^0-9.]" "" CMAKE_SYSTEM_VERSION "${CMAKE_SYSTEM_VERSION}")
+if("${CMAKE_SYSTEM_VERSION}" VERSION_LESS "7.0.0")
+    message(FATAL_ERROR "This toolchain file is not supported for QNX ${CMAKE_SYSTEM_VERSION}")
+else()
+    message(STATUS "QNX Version is ${CMAKE_SYSTEM_VERSION}")
+endif()
+
+# Since we are passing while crosscompilation CMAKE_FIND_ROOT_PATH(containing QNX third party dependencies)
+# as command line argument to cmake we have to take it into account when setting CMAKE_FIND_ROOT_PATH
+# variable below.
+# ${QNX_TARGET}/usr contains platform independent headers.
+# But QNX_TARGET is a root directory containing sysroots for different0 platforms.
```

Comment:
@BSolonenko typo

Reply:
@AByzhynar fixed in https://github.com/Ford-Luxoft/sdl_core/commit/a3a73a446d1a8fe30f7b98dde719b35719b6d731